### PR TITLE
Use cc to find includes

### LINKED ISSUE #11: Unable to find windows.h
Well, on my machine it builds fine for `gnu` builds, however if i run cargo build under plain cmd.exe console, it can find "rc.exe" but can't find `windows.h`.  After "vcvarsall.bat" is run everything goes fine.

So i think maybe this library should try to detect and set up the environment in certain cases?

### PR DESCRIPTION
**Pull request by roblabla**
Fixes #11

### CODE REVIEW

File: src/windows_msvc.rs [hunk 1]
```diff
@@ -141,24 +141,11 @@ fn include_windows_10_kits(kit_root: &str) {
 }
 
 fn include_windows_10_kits_impl(kit_root: &str) {
-    const VAR_INCLUDE: &str = "INCLUDE";
-
-    if let Ok(include_root) = fs::read_dir(kit_root.to_string() + r"\Include\") {
-        let mut include = env::var(VAR_INCLUDE).unwrap_or_default();
```
Comment (nabijaczleweli) [inline src/windows_msvc.rs:147]: Before I even start to think about reviewing this mess, is this branch even entered on your system? If so, does the final env var have any elements in it?

Comment (roblabla) [reply src/windows_msvc.rs:147]: I mean, I'm using this in a closed source project, and it fixed the underlying issue that winuser.h doesn't get found.

I dumped the include path being set, here it is:

`C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\include;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\ucrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\um;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\cppwinrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\winrt;C:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\shared;`

**Comment by nabijaczleweli** [comment]
I've updated this slightly to append `cc`'s include path to the one found as before, instead of replacing it; would you mind trying the PR HEAD?

**Comment by roblabla** [comment]
Yup, just tested it, it works without problems!

**Comment by nabijaczleweli** [comment]
Ace, released in [`v1.6.1`](https://github.com/nabijaczleweli/rust-embed-resource/releases/tag/v1.6.1), then. Cheers!

**Pull request merged**